images/linux/scripts/installers/go.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+################################################################################
+##  File:  go.sh
+##  Desc:  Installs go, configures GOROOT, and adds go to the path
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+golangTags="/tmp/golang_tags.json"
+# This function installs Go using the specified arguments:
+#   $1=MajorVersion (1.11)
+#   $2=IsDefaultVersion (true or false)
+function InstallGo () {
+    version=$( getFullGoVersion $1 )
+    echo "Install Go $version"
+    curl -sL https://dl.google.com/go/$version.linux-amd64.tar.gz -o $version.linux-amd64.tar.gz
```

Comment:
`curl -sL https://dl.google.com/go/$version.linux-amd64.tar.gz`
I would suggest to use `${version}` approach in such lines: `curl -sL https://dl.google.com/go/${version}.linux-amd64.tar.gz` for readability

Reply:
Done.